plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/TestElasticsearchBackpressure.java

```diff
@@ -50,7 +50,9 @@ protected QueryRunner createQueryRunner()
                 ImmutableMap.of(),
                 // This test can only run on a single node, otherwise each node exports its own stats beans and they override each other
                 // You can only bind one such bean per JVM, so this causes problems with statistics being 0 despite backpressure handling
-                1);
+                1,
+                // Use a unique catalog name to prevent other tests from overriding stats bean
```

Comment:
How does it work? From what I see in `ElasticsearchConnectorModule` catalog name is not even used.

Reply:
It is used in `ElasticsearchConnectorFactory` to create an instance of `ConnectorObjectNameGeneratorModule`, where it used instantiate `ConnectorObjectNameGenerator`. 
This generator uses the `catalogName` as a parameter in the generated bean name, so the generate name will be like `trino.plugin.elasticsearch.client:name=elasticsearch-backpressure,type=elasticsearchclient`